src/content/docs/accounts/accounts-billing/account-structure/new-relic-account-structure.mdx

```diff
@@ -1,34 +1,43 @@
-* New Relic One account/user model: This is our new, improved model. We currently are migrating some customer accounts to this model and eventually all our customers will be on this model. To learn more and see if you can migrate your users, see [Migrate users](/docs/accounts/original-accounts-billing/original-users-roles/user-migration). To learn more about how account and user management works, keep reading. 
+## New Relic One account/user model [#new-model]
 
-## Account structure for New Relic One account/user model [#account-structure]
+<Callout variant="important">
+This section is about organizations on the New Relic One account/user model, not the original model. [Learn more about the difference.](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models) 
+</Callout>
 
-For organizations on the New Relic One account/user model, here are some details about how organization and account management work: 
+At New Relic, an "organization" is the representation of a New Relic customer. The organization contains everything pertinent to a New Relic customer: its accounts, its users, and its data. For organizations on [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/), usage that impacts billing is aggregated at the organization level. 
 
-* The [Standard edition](https://newrelic.com/pricing) allows for a single account per organization. [Pro and Enterprise editions](https://newrelic.com/pricing) allows for [multiple accounts](/docs/accounts/accounts-billing/account-structure/mastersub-account-structure/#users).
-* At the top level of a New Relic account hierarchy is the concept of the "organization." Your group of related accounts that share the same billing are inside a single "New Relic organization." 
-* Currently you can't add accounts to your organization on your own. To add accounts, talk to your New Relic account representative. 
-* To learn how you assign users permissions and access to specific accounts, see [Manage users](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts). 
+A New Relic "account" can be considered a workspace. For example, you might have an account for a specific app, or a set of related hosts and services for a specific initiative or project, or you might have an account for a specific team.
 
-Having trouble accessing a specific account or feature? See [Factors affecting access](/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data). 
+Our [Standard edition](https://newrelic.com/pricing) allows for a single account per organization. [Pro and Enterprise editions](https://newrelic.com/pricing) allow for multiple accounts per organizations. Currently you can't add accounts to your organization on your own. To add accounts, talk to your New Relic account representative. 
```

Comment:
It feels weird to leave it at "standard edition". Standard edition... what? Pricing?

There's a strange jump in context from the paragraph prior to this one.

It might feel like I'm being nitpicky about words, but this stuff is tough to understand and wherever we can simplify or clarify language, it seems worth doing.